apps/src/aiTutor/views/AssistantMessageFeedback.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {useState} from 'react';
+import Button, {buttonColors} from '@cdo/apps/componentLibrary/button/Button';
+
+import {saveFeedback, FeedbackData} from '../interactionsApi';
+import style from './chat-workspace.module.scss';
+
+interface AssistantMessageProps {
+  messageId: number | undefined;
+}
+
+const AssistantMessageFeedback: React.FC<AssistantMessageProps> = ({
+  messageId,
+}) => {
+  const [feedbackState, setFeedbackState] = useState<FeedbackData>({
+    thumbsUp: false,
+    thumbsDown: false,
+  });
+
+  const handleFeedbackSubmission = async (
```

Comment:
If we want this to be more generalizable in the future, we could pass `handleFeedbackSubmission` as a callback and the checked state as a prop, but that seems like work for a future PR.